index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio Project</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+   <header>
+    <nav class="navbar">
+        <div class="logo-div">
+            <p class="logo">My Logo</p>
+            <img src="images/Union.png" alt="image" class="nav-image">
+        </div>
+        <div class="navbar-items">
+            <ul>
+                <li><a href="#" class="portfolio">Portfolio</a></li>
+                <li><a href="#" class="about">About</a></li>
+                <li><a href="#" class="contact">Contact</a></li>
+                </ul>
+        </div>
+    </nav>
+</header> 
+    <section class="wrapper">
+        <article class="intro">
+            <h2 class="h2-intro">I'm Eligrand<br/>
+              Glad to see<br />
+               you!
+            </h2>
+                <p class="p-intro">
+                    I’m a software developer! I can help you build a product , 
+                    feature or website Look through 
+                    some of my work and experience! If you like what you see and have a project you need
+                     coded, don’t hesitate to contact me.
+                </p>
+                
+            <div class="connect">
+                <p class="let-connect">LET'S CONNECT</p>
+                <div class="social-media">
+                    <a href="#" class="twitter"><img src="images/twitter.png" alt="twitter-icon"></a>
+                    <a href="#" class="linkedin"><img src="images/Linkedinicon.svg" alt="linkedin-icon"></a>
+                    <a href="#" class="median"><img src="images/median.svg" alt="median-icon"></a>
+                    <a href="#" class="github"><img src="images/github.png" alt="github-icon"></a>
+                    <a href="#" class="peace"><img src="images/peace.svg" alt="peace-icon"></a>
+                  </div>
```

Comment:
Good job giving a link for each social icon! 👍🏼 

- [ ] Please as this is a list of items, for better practice wrap the list of items on a `ul` and `li`. So in the end, you will have instead a `div` (for better readability avoid using div everywhere) you will have a `ul` and inside them a `li` for each `a` that you have already created. This will give you a better structure on the document 🥇